<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test</class>
 <widget class="QMainWindow" name="test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>窗口</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>640</width>
      <height>512</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>612</width>
      <height>512</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="selectPathButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openImageButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>选择基准图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_path">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>路径: （未选择）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_filename">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>60</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>文件: （未选择）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadIntrinsicsButton">
 <property name="geometry">
  <rect>
   <x>700</x>
   <y>100</y>
   <width>100</width>
   <height>30</height>
  </rect>
 </property>
 <property name="text">
  <string>导入相机内参</string>
 </property>
</widget>
<widget class="QPushButton" name="undistortButton">
 <property name="geometry">
  <rect>
   <x>700</x>
   <y>140</y>
   <width>100</width>
   <height>30</height>
  </rect>
 </property>
 <property name="text">
  <string>畸变矫正</string>
 </property>
</widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
